### ISSUE TITLE
Can not generate Request Param swagger config

### ISSUE THREAD

**Issue by cuteQ** [parvez3019/go-swagger3#29]
In Param config, if we pass {in} as query, path, it won't generate Request Param Config in oas.yml.
It also doesn't have {required} information in the oas.yml.
Does someone know how to generate it or it's a bug?

**Comment by cuteQ** [comment]
This happened to the Params which are objects in request

**Comment by qequ** [comment]
Hi @cuteQ thanks for the report, can you pass an example to replicate it?

**Comment by cuteQ** [comment]
Yes, when the Params is an object, then this request params won't be generated:

type SwaggerTestRequestSchema struct {
	AppId string `query:"app_id,required"`
}

type SwaggerTestResponseSchema struct {
	ResponseMessage string `json:"response_message,omitempty"`
}

// @Title		        Get hello world information
// @Description	Description content
// @ID		         user-info
// @Param		swaggerRequest    query    SwaggerTestRequestSchema  true "SwaggerTestRequestSchema JSON"
// @Success		200			       object   SwaggerTestResponseSchema "SwaggerTestResponseSchema JSON"
// @Failure		500			       object   SwaggerTestResponseSchema  "SwaggerTestResponseSchema JSON"
// @Router		/open_api/v1.3/swagger/test/    [post]
func SwaggerTestViewHandler(ctx context.Context, reqCtx *app.RequestContext) {
}

**Comment by aveyuan** [comment]
Hi，I also same question, can you run it ?

```go
package main


// @Title get hello
// @Description get hello
// @Success  200  object  UsersResponse 
// @Failure  400  object  ErrorResponse 
// @Param Users query Users false "Group ID"
// @Route /api/group/{groupID}/users [get]
func Helloworld() {
	
}

type Users struct {
	ID   uint64 `json:"id" example:"100" description:"ID"`
	Name string `json:"name" example:"name"`
}

type UsersResponse struct {
	Data []Users `json:"users" example:"[{\"id\":100, \"name\":\"Parvez\"}]"`
}

type Error struct {
	Code string `json:"code"`
	Msg  string `json:"msg" skip:"true"`
	// use skip:"true" if you want to skip the field in the documentation spec
}

type ErrorResponse struct {
	ErrorInfo Error `json:"error"`
}

// @Version 1.0.0
// @Title Backend API
// @Description API usually works as expected. But sometimes its not true.

func main() {
	
}

```

**Comment by aveyuan** [comment]
I see,  must be in model. package...

<img width="1109" height="449" alt="Image" src="https://github.com/user-attachments/assets/c6098fc9-872b-4912-8401-b9fae56bec13" />
